src/NuGet.Core/NuGet.Packaging/Signing/Archive/SignedPackageArchiveUtility.cs

```diff
@@ -104,7 +104,9 @@ public static Stream OpenPackageSignatureFileStream(BinaryReader reader)
             var buffer = new byte[localFileHeader.UncompressedSize];
 
             reader.BaseStream.Seek(offsetToData, SeekOrigin.Begin);
+#pragma warning disable CA2022 // Avoid inexact read
```

Comment:
It looks like a false positive in CA2022 analyzer has been fixed in https://github.com/dotnet/roslyn-analyzers/issues/7268. Do we still need these suppressions?

Reply:
I'm not sure. False positive fix is for `SystemIOMemoryStream`, `SystemIOUnmanagedMemoryStream` only.

Reply:
I changed the line at [https://github.com/NuGet/NuGet.Client/blob/dev/build/DotNetSdkVersions.txt#L2](https://github.com/NuGet/NuGet.Client/blob/dev/build/DotNetSdkVersions.txt#L2) to `-Channel 9.0 -Quality daily` and then executed the [configure.ps1](https://github.com/NuGet/NuGet.Client/blob/dev/configure.ps1) script locally. This change resulted in the installation of the `9.0.100-preview.4.24209.7` .NET SDK. After trying to build the repository locally, I noticed no CA2022 errors.

Reply:
That's good. The problem is that we need pick up a new SDK in VMR to avoid this errors. We will re-bootstrap after Preview 4 ships, in May.

Reply:
> We will re-bootstrap after Preview 4 ships, in May.

Does this mean that we should consider this PR now and revert the changes once Preview 4 ships in May?